src/resources/resource_paths.c

```diff
@@ -91,9 +91,14 @@ static bool scan_potential_resource_dirs(path *result, const str *src, const cha
     vector_iter_begin(&paths, &it);
     foreach(it, slice) {
         path_from_parts(&test, str_c(slice), find);
+        if(!path_resolve(&test)) {
+            log_warn("Unable to resolve %s; does it exist?", path_c(&test));
+            continue;
+        }
         log_debug("Looking for resources in %s ...", path_c(&test));
         if(path_is_file(&test)) {
             path_from_parts(result, str_c(slice), append);
+            path_resolve(result);
```

Comment:
The two calls to path_resolve is a (probably innocuous) TOCTOU bug.

If we make `find` a relative path from `append`, then we can call path_resolve once on "slice..find" and check for the existence of openomf.bk inside of that canonicalized path.